### DISCUSSION TITLE
Scroll didn't restore to the top after transition to new page

### DISCUSSION THREAD

**Discussion by evgeniyworkbel** [vercel/next.js#59121] [Help]
### Summary

We have home page (`/`) and portfolio cases (`/portfolio/[id]`).

Our steps to get issue:
- we at home page
- scroll home page
- stop at some case and click it
- scroll didn't restore to the top - we want to see hero, but scroll is located lower than it's needed

This also happens with not nesting routes, but static like `/about`
Why does it occur and how to fix it?
We use next.js <Link> without extra attributes, only `href` and `classname`

https://github.com/vercel/next.js/assets/93836126/b3877cbc-f3cc-4913-a10f-2e1a5bc29379

### Additional information

```js
Next.js v13.4.19, we use App Router
```

**Comment by alilland** [comment]
ever find a solution to this? 🙁

**Reply by evgeniyworkbel** [reply]
this was fixed in one of version, i dont remember whcih in

**Comment by Arunjenson-ss** [comment]
any update on this?

**Reply by evgeniyworkbel** [reply]
this was fixed in one of version, i dont remember whcih in
